registry/txt.go

```diff
@@ -322,19 +318,25 @@ type affixNameMapper struct {
 	prefix              string
 	suffix              string
 	wildcardReplacement string
+	managedRecordTypes  []string
 }
 
 var _ nameMapper = affixNameMapper{}
 
-func newaffixNameMapper(prefix, suffix, wildcardReplacement string) affixNameMapper {
-	return affixNameMapper{prefix: strings.ToLower(prefix), suffix: strings.ToLower(suffix), wildcardReplacement: strings.ToLower(wildcardReplacement)}
+func newaffixNameMapper(prefix, suffix, wildcardReplacement string, managedRecordTypes []string) affixNameMapper {
+	return affixNameMapper{
+		prefix:              strings.ToLower(prefix),
+		suffix:              strings.ToLower(suffix),
+		wildcardReplacement: strings.ToLower(wildcardReplacement),
+		managedRecordTypes:  managedRecordTypes,
+	}
 }
 
 // extractRecordTypeDefaultPosition extracts record type from the default position
 // when not using '%{record_type}' in the prefix/suffix
-func extractRecordTypeDefaultPosition(name string) (baseName, recordType string) {
+func (pr affixNameMapper) extractRecordTypeDefaultPosition(name string) (baseName, recordType string) {
```

Comment:
```suggestion
func (anm affixNameMapper) extractRecordTypeDefaultPosition(name string) (baseName, recordType string) {
```

Reply:
Well, `affixNameMapper` already has a lot of methods and all of them use the identifier `pr`.

Reply:
Shall I change it?

Reply:
That would be great, yes :+1:

Reply:
Done